activerecord/test/cases/adapter_test.rb

```diff
@@ -514,6 +517,8 @@ def test_truncate_tables
       assert_equal 0, Post.count
       assert_equal 0, Author.count
       assert_equal 0, AuthorAddress.count
+    ensure
+      reset_fixtures('posts', 'authors', 'author_addresses')
```

Comment:
I'm not sure why codeclimate raises no offence, but we have `.rubocop.yml` and this violates the `Style/StringLiterals` cop. cc6bff3daa4da4d2c6935761d0ab059201f772e2

https://github.com/rails/rails/blob/cc6bff3daa4da4d2c6935761d0ab059201f772e2/.rubocop.yml#L165-L167

Reply:
🙇 thanks for following up and fixing that for me ❤️